CoreIntegrationTests/Infrastructure/MongoDbDocumentTestBase.Main.cs

```diff
@@ -1,19 +1,17 @@
-﻿using MongoDbGenericRepository;
-using MongoDbGenericRepository.Models;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
 using System.Linq.Expressions;
 using System.Runtime.CompilerServices;
 using System.Threading;
 using System.Threading.Tasks;
+using MongoDbGenericRepository.Models;
 using Xunit;
 
 namespace CoreIntegrationTests.Infrastructure
 {
-    public abstract partial class MongoDbDocumentTestBase<T> :
-        IClassFixture<MongoDbTestFixture<T, Guid>>
```

Comment:
MongoDbDocumentTestBase<T> is a partial class split between MongoDbDocumentTestBase.Main.cs and MongoDbDocumentTestBase.Update.cs - both files had the interface beign implemented in it making one of them redundant - this was the one that had the warning for the redundancy it so it got removed. happy to add it back or move the interface here instead of in MongoDbDocumentTestBase.Update.cs